www/search.spt

```diff
@@ -1,16 +1,20 @@
-from gratipay.utils import markdown, excerpt_intro
+from gratipay.utils import markdown, truncate
 from gratipay.utils.i18n import LANGUAGES_2, SEARCH_CONFS, strip_accents
 from markupsafe import Markup
 [---]
 
 def process_excerpt(excerpt):
-    excerpt = excerpt_intro(markdown.render(excerpt), 256, '')
-    if excerpt:
-        excerpt = Markup('&middot; ') + excerpt
-    return excerpt
+    """Take a statement excerpt from *some*where and process it.
+    """
+    if not excerpt:
+        return Markup('')
+    return Markup('&middot; ' + excerpt)
 
 def get_processed_excerpt(participant):
-    statement = participant.get_statement(request.accept_langs)[0] or ''
+    """For a username match, take the first part of the statement.
+    """
+    statement = participant.get_statement(request.accept_langs, scrubbed=True)[0] or ''
+    statement = truncate(statement, 128, '')
```

Comment:
128 is plenty to fill the UI slot. No reason to send more than we're ever going to display.